### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by ArshiaSaleem98**
### Type:

- [ ] refactor: enhancements.
 
updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +64,7 @@ http://guidesmiths-react-form-builder.s3-website.eu-central-1.amazonaws.com/
 |   onSubmit*    |   Action to be realised "onSubmit" form    |    function       |   -    |
 |   language	| Shortcut with the language  to render components in multiple languages (`country`,`date`) <br /> <br /> Available laguages: `es`,`de`,`fr`,`en`  	| string   	|   en	|
 |  isoCode      |   Isocode of the country to show as default in phone input |  string     | GB
+|  isMobile      |   Checked component by default | boolean    | false
```
Comment (ismaelocaramelo) [inline README.md:67]: may be we can change it to have a better description? Something more specific of the flag. E.g:
`A boolean flag to determine if the we're from a mobile port view`

File: README.md [hunk 2]
```diff
@@ -81,6 +89,7 @@ http://guidesmiths-react-form-builder.s3-website.eu-central-1.amazonaws.com/
 |   name*	|  Checkbox name  	|  string 	|  - 	|
 |   type*   | Must be `checkbox`| string | - |
 |   label	|  Text shown next to the checkbox. This text can be written in markdown style 	|  string  	|   ''	|
+|   isFullWidth   |   Define if the field takes all the available width in the grid or not   |    boolean       |   false   |
```
Comment (ismaelocaramelo) [inline README.md:92]: Nice :)

File: README.md [hunk 3]
```diff
@@ -117,9 +126,13 @@ https://user-images.githubusercontent.com/79102959/134894112-e4f38ced-0992-428c-
 |   name*	|  Country component name  	|  string 	|  - 	|
 |   type*   | Must be `country` | string | - |
 |   label	|  Text shown over the coutnry list|  string  	|   ''	|
+|   language	| An string with the language shortcode in which you want to display the country names opt:  <br /> Available laguages: `es`,`de`,`fr`,`en`  	| string   	|   en	|
 |   placeholder    |   Placeholder displayed in the select    |    string       |   ''   |
 |   priorityOptions    |   Array of strings with shortcode(s) of the countries that want to be displayed first in the countries list. Ex: ['GB', 'ES']    |    string       |   '' |
 |   returnCountryName	|  Returns the country name (in English) as each option value, instead of the country code  	|  boolean 	|  false	|
+|   **countryAndRegionsData**	| An object or array of objects with the acronym(s) and the names of the countries that you want to be shown in the select.  	| json   	|  [] 	|
+|  countryName     |   The name of the country |  string     | ''
+|  countryShortCode     |   The name of the country |  string     | ''
```
Comment (ismaelocaramelo) [inline README.md:135]: Probably a better description should be added here.` Country short code ` for instance

File: README.md [hunk 4]
```diff
@@ -144,6 +157,42 @@ Reminder: the 'countryAndRegions' prop that can be sent in the ReactFormBuilder
 }
 ```
 
+### Custom list country example:
```
Comment (ismaelocaramelo) [inline README.md:160]: 👌

**Review by ismaelocaramelo — CHANGES_REQUESTED** [review]
Nice update @ArshiaSaleem98 , I've just left some comments

**Comment by ArshiaSaleem98** [comment]
Got your points @ismaelocaramelo  i will update it. thanks.

**Pull request closed without merging**